src/libs/ReportUtils.ts

```diff
@@ -1422,7 +1422,7 @@ function isOneTransactionReport(reportID: string): boolean {
 function isOneTransactionThread(reportID: string, parentReportID: string): boolean {
     const parentReportActions = allReportActions?.[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${parentReportID}`] ?? ([] as ReportAction[]);
     const transactionThreadReportID = ReportActionsUtils.getOneTransactionThreadReportID(parentReportID, parentReportActions);
-    return reportID === transactionThreadReportID;
+    return reportID?.toString() === transactionThreadReportID?.toString();
```

Comment:
Shouldn't both of these already be strings?

Reply:
Hmm I've asked for clarification [here](https://github.com/Expensify/App/issues/43238#issuecomment-2162625565) on what's best to do. Ideally this would be fixed in the BE first, but of course that can take some time.

Reply:
@tienifr Could you respond to [this](https://github.com/Expensify/App/issues/43238#issuecomment-2163235957) comment?

Reply:
I put up a PR to fix the type on the BE. We should be able to remove the cast to string once that's merged/deployed